react/src/App.tsx

```diff
@@ -82,6 +82,15 @@ function App() {
           <div className="sdds-headline-02 sdds-u-pb1">Checkbox</div>
           <sdds-checkbox>Checkbox</sdds-checkbox>
         </div>
+        <div>
+          <div className="sdds-headline-02 sdds-u-pb1">Chip</div>
+          <button className="sdds-chip">
+            <span className="sdds-chip-text">Chip text</span>
+          </button>
+          <button className="sdds-chip sdds-chip-active">
+            <span className="sdds-chip-text">Chip text</span>
+          </button>
```

Comment:
Maybe add one with an icon too (web-component icon) so we test that one too.

Reply:
Added one with the icon. It does not look as expected. But I say we create a ticket for it and move on for now :)

Reply:
Did some minor investigating just now and looks like it the name issue as with className on webcomponents. So I still say we move on for now and get back to it after dark/light mode tests are done

Reply:
Absolutely, I agree!